### ISSUE TITLE
New god: dream god

### ISSUE THREAD

**Issue by mciul** [i7/kerkerkruip#418]
The dream god reduces bad effects of dreams, either by decreasing penalties or changing effects like telling you which chest to open in Monty Hall or putting Malygris to sleep in the Dream of Sleeping.

Intervention puts enemies to sleep (or just does a Morphean grenade effect)

**Comment by VictorGijsbers** [comment]
Obviously, the dream God needs to be called Morpheus. :-) I think he should reduce the negative effects of dreams but not eliminate them entirely. Also, we need more dreams to make this fun.

**Comment by mciul** [comment]
I agree!
